docs/user-guide/configuration/workflow.md

```diff
@@ -159,6 +163,8 @@ jobs:
             - echo: echo staging pr
 ```
 
+See the [branch filtering example repo](https://github.com/screwdriver-cd-test/branch-filtering-example) for reference. To see how branch filtering works with pull requests, see our [example pull request](https://github.com/screwdriver-cd-test/branch-filtering-example/pull/2).
```

Comment:
any additional clarification regarding PR workflow based on committing branch (branch PR against) instead of pipeline branch?

Reply:
not that I am aware of
any ideas?

Reply:
Say something like the PR to a branch will follow the workflow pattern indicated by that branch’a screwdriver.yaml.

Reply:
Updated